README.adoc

```diff
@@ -2,25 +2,151 @@
 This guide walks you through an overview of Spring Cloud Stream and the process of creating an event-driven
 streaming application.
 
+== Starting with Spring Initializr
+
+You can use this https://start.spring.io/#!type=maven-project&language=java&platformVersion=3.0.1&packaging=jar&jvmVersion=17&groupId=com.example.stream&artifactId=name-sink&name=name-sink&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.stream.name-sink&dependencies=cloud-stream,amqp[pre-initialized project] and click *Generate* to download a ZIP file. This project is configured to fit the examples in this tutorial.
+
+To manually initialize the project:
+
+. Navigate to https://start.spring.io.
+This service pulls in all the dependencies you need for an application and does most of the setup for you.
+. Choose either Gradle or Maven and the language you want to use. This guide assumes that you chose Java.
+. Click *Dependencies* and select _Cloud Stream_ and _Spring for Rabbit MQ_.
+. Click *Generate*.
+. Download the resulting ZIP file, which is an archive of a task application that is configured with your choices.
+
+NOTE: If your IDE has the Spring Initializr integration, you can complete this process from your IDE.
+
+NOTE: You can also fork the project from GitHub and open it in your IDE or other editor.
+
 === What is Spring Cloud Stream?
 A framework for building event-driven Spring Boot microservices for real-time stream processing. You can learn more about
 the framework from the link:https://spring.io/projects/spring-cloud-stream[project-site],
 link:https://spring.io/projects/spring-cloud-stream#learn[documentation],
 and link:https://github.com/spring-cloud/spring-cloud-stream-samples[samples].
 
-=== Stream Processing with Apache Kafka
-In the following guide, we develop three Spring Boot applications that use Spring Cloud Stream's support for Apache Kafka and deploy
-them to Cloud Foundry, to Kubernetes, and on your local machine.
+=== Stream Processing Application Overview
+
+In this guide, we will demonstrate the capability of Spring Cloud Stream.  We will create 3 applications to showcase the different
```

Comment:
Change to:

This guide demonstrates the capability of Spring Cloud Stream.  We create three applications to showcase the different